sysutils/ntpsec/Portfile

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           waf 1.0
+
+name                ntpsec
+version             0.9.6
+categories          sysutils net
+maintainers         {schenkel.net:leonardo @lbschenkel} openmaintainer
+description         A secure, hardened, and improved implementation of NTP
+license             BSD-2-Clause BSD-3-Clause CC-BY-4.0 NTP MIT
+platforms           darwin
+long_description    A secure, hardened, and improved implementation of Network \
+                    Time Protocol derived from NTP Classic, Dave Mills’s original.
+homepage            https://www.ntpsec.org/
+
+conflicts           ntp openntpd
+
+master_sites        ftp://ftp.ntpsec.org/pub/releases/
+checksums           rmd160  9527cc6bc78f1a02c868fbb6a1fb583461b8fa81 \
+                    sha256  9d15ac59b67911eb1b42fe5916e8d00ded8b71ba34f1ab4fc0f251fc0d90d5e5
+
+depends_build       port:bison
+depends_lib         path:lib/libssl.dylib:openssl port:python27
+
+patchfiles          patch-macros.diff patch-smear.diff
+patch.pre_args      -p1
+
+set pythondir       "${frameworks_dir}/Python.framework/Versions/2.7/lib/python2.7/site-packages"
```

Comment:
I have changed the port to use the python port group, I hope it is slightly cleaner now.